raiden/api/rest.py

```diff
@@ -1733,11 +1733,15 @@ def settlement_light(self,
                 channel_identifier=channel_identifier,
                 signed_settle_tx=signed_settle_tx
             )
+            return self.update_channel_state(registry_address, channel_state)
+        except ChannelNotFound as e:
+            return api_error(
+                errors="Channel is already unlocked.",
```

Comment:
here i would say `Channel is already settled` instead, and omit the `.` to be consistent with the previous error messages.